health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/bill_inbox/bill_boundary_filter.js

```
@@ -0,0 +1,135 @@
+                        display: "flex",
+                        flexDirection: "row",
+                        justifyContent: "space-between",
+                    }}
+                >
+                    <div style={{ display: "flex", alignItems: "center" }}>
+                        <SVG.FilterAlt width={"32px"} height={"32px"} fill={"#c84c0e"} />
+                        <span className="custom-inbox-filter-heading">{t("HCM_AM_FILTER")}</span>
+                    </div>
+
+                    <span onClick={() => { }} style={{ border: "1px solid #e0e0e0", padding: "6px", marginBottom: "10px" }}>
+                        <svg width="17" height="17" viewBox="0 0 16 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)